lib/Alchemy/Phrasea/Collection/Reference/DbalCollectionReferenceRepository.php

```diff
@@ -99,6 +112,29 @@ public function findByCollectionId($databoxId, $collectionId)
         return null;
     }
 
+    public function findHavingOrderMaster(array $subset = null)
```

Comment:
It would be nicer to call this parameter collectionSubset for better readibility